content/en/docs/components/training/job-scheduling.md

```diff
@@ -87,6 +89,23 @@ configure operator to select the scheduler name for gang-scheduling in the follo
 ...
 ```
 
+- mpi-operator
+
+```diff
+...
+    spec:
+      containers:
+      - args:
++       - --gang-scheduling
++       - volcano
```

Comment:
This should also work, right?

```suggestion
+       - --gang-scheduling=volcano
```

Reply:
Yes, you're right. 
I used a space delimiter, not an equal delimiter since we use a space delimiter in L102-L103.

Reply:
Done.